### DISCUSSION TITLE
macs2 not available

### DISCUSSION THREAD

**Discussion by drighelli** [stuart-lab/signac#1628] [Q&A] [answered]
Hi, 

I'm using signac to analyze multiple 10x multiome samples, but I have some problems with macs2 on the server.

Is there a way to use signac without macs2 calling peaks?

Thanks!

**Comment by timoast** [comment, marked as answer]
You don't need to use macs2, you can use any method to get a peak set. An example of another approach is shown in the merge vignette: https://stuartlab.org/signac/articles/merging

**Reply by drighelli** [reply]
thanks!
